src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1457,6 +1459,15 @@ The `cols` keyword argument determines the columns of the returned data frame:
 * A vector of `Symbol`s or strings: only listed columns are kept.
   Columns not present in some data frames are filled with `missing` where necessary.
 
+The `source` keyword argument, if not `nothing` (the default) specifies the additional
+column to be added as a first column in the resulting data frame that will identify
+the source data frame. In this case `source` can be a `Symbol` or an `AbstractString`
+in which case the identifier will be the number of the passed source data frame.
+Alternatively `source` can be a `Pair` consisting of a `Symbol` or an `AbstractString`
+and a vector specifying the data frame identifiers (the identifiers do not have to be
+unique). The name of the source column name is not allowed to be present in any
+source data frame.
```

Comment:
```suggestion
The `source` keyword argument, if not `nothing` (the default), specifies the additional
column to be added in the first position in the resulting data frame that will identify
the source data frame. It can be a `Symbol` or an `AbstractString`,
in which case the identifier will be the number of the passed source data frame, or
a `Pair` consisting of a `Symbol` or an `AbstractString`
and of a vector specifying the data frame identifiers (which do not have to be unique).
The name of the source column is not allowed to be present in any
source data frame.
```